modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -44,19 +46,22 @@ All the rulesets are disabled by default. For each rule, you can: +
 * *Detect and allow violations* - The violation is detected and you will get information, per incident, in your log at the `INFO` level.
 * *Detect and reject violations* - The request is rejected and returns a response status of `HTTP/1.1 400 BAD REQUEST - web application firewall` and information about the violation is also sent to the log at INFO level per incident.
 . In *Advanced Performance Options*, select from the following options: +
-* *Disable body scanner* - This option works the same as it does for the request ruleset, however, there are some additional important caveats:
+* *Disable body scanner* - By default, the response body is scanned unless the body is larger than 1 MB, in which case the scan is skipped. +
+Check this option if response body scanning isn't needed, or to reduce CPU consumption. +
+This option works the same as it does for the request ruleset with some additional important caveats:
 +
 [IMPORTANT]
-If a body is larger than 1 MB, the body scan is skipped, even if you selected *Disable body scanner*. 
+If a body is larger than 1 MB, the body scan is skipped, even if you do not select *Disable body scanner*. 
 +
 * Response messages must have one of the following MIME types in the content type header, or the WAF rules will not be evaluated:
```

Comment:
In GitHub View file, this shows up at the same level as the bullets, making it look like a third option. Not clear what it goes with.